guides/concepts/multi-sig.md

```diff
@@ -144,3 +148,4 @@ Source account setup:
   medium threshold: 0
   high threshold: 0
 ```
+Note that even though the thresholds are 0 here, the master key cannot successfully sign a transaction because it's own weight is 0 (this is the exception to the equal to or greater than rule).
```

Comment:
This is true indeed and it got me thinking if this should be changed in stellar-core. To make this example less confusing, maybe we can change it to:
```
   master key weight: 0
   low threshold: 1
   medium threshold: 1
   high threshold: 1
```
The same effect will be achieved but without any exceptions.

Reply:
I think the exception will still exist in stellar-core, which is why I explicitly listed it out as an exception case here to let users know how it would behave in such a scenario.

I think it would be better if we set the threshold range to be from `1-255` instead of `0-255`, which would remove this exception case, if we updated all accounts to use this new threshold range *(change all 0 thresholds to 1)* it would not introduce any regressions since in theory, the behavior should stay the same.

maybe this can be discussed by the core team?

Reply:
Yeah, I think you can start a discussion about it in [stellar-protocol](https://github.com/stellar/stellar-protocol) because it would be a protocol change.

Reply:
yeah, filed an issue to discuss: https://github.com/stellar/stellar-protocol/issues/54